README.md

```diff
@@ -1,31 +1,181 @@
-# IoT Reference Integration on STM32U5 Arm Cortex-M33 MCU
+# Getting Started Guide for STM32U5 IoT Discovery Kit with AWS 
 
+## Overview 
 
-### Perfroming Over-the-air (OTA) Firmware Update for [B-U585I-IOT02A discovery kit](https://www.st.com/en/evaluation-tools/b-u585i-iot02a.html)
+The STM32U585 IoT Discovery kit comes with 2048 KB of Flash memory, 786 kB of RAM and is based on Arm® Cortex®-M33.   
 
-The project shows an IoT reference implementation of how to integrate FreeRTOS libraries on STM32U5 platform to perform OTA update with AWS IoT using both trustzone and non-trustzone hardware capablities. The demo runs FreeRTOS OTA agent as one of the RTOS tasks in background, which waits for OTA updates from cloud. The non-trustzone version of the demo leverages internal flash memory's dual bank. The total 2MB internal flash is split into two banks of 1MB each. The main firmware is running on one bank, while downloading new firmware image into another bank. Thus, the total firmware image size should not exceed 1MB.
+The STM32U5 IoT Discovery Kit is equipped with a Wi-Fi and Bluetooth module, microphones, a temperature and humidity sensor, a magnetometer, an accelerometer and gyroscope, a pressure sensor, as well as Time-of-Flight (ToF) and gesture-detection sensors. 
 
-Please follow the below steps to bring up the demo project and perform OTA update:
+The board also comes with 512-Mbit octal-SPI Flash memory, 64-Mbit octal-SPI PSRAM, 256-Kbit I2C EEPROM, as well as ARDUINO Uno V3, STMod+, and Pmod expansion connectors, plus an expansion connector for a camera module, and STLink-V3E embedded debugger. 
 
-#### Pre-requisites
+## Hardware Description 
 
-* AWS CLI
-  * The steps below uses AWS CLI commands to perform interactions with cloud,
-  * AWS CLI can be installed using this [link](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
-  * Run `aws configure` and set the access Key ID, secret Key and region for your AWS account
+ https://www.st.com/en/microcontrollers-microprocessors/stm32u5-series.html 
+ 
+ ###  User Provided items
 
-* If not created already, provision a new thing name and have the required certificates and policies attached to connect to AWS IoT broker.
+ 
+ The User must provide: 
+A USB cable (standard A port to micro-B port) 
 
```

Comment:
Suggest to add Folder Structure to help users navigate the code/repository

Reply:
Folder structure is provided in the Set up your Development Environment Section. Would you also prefer it be added here ?